src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethod3.java

```diff
@@ -20,6 +19,7 @@ public class InputJavadocMethod3 implements Input {
      * (even on platforms with a different line separator).
      * For header lines containing {@code "\n\n"} checkstyle will forcefully
      * expect an empty line to exist. See examples below.
+
```

Comment:
Please revert extra changes, here and in other files.

Reply:
Apologies for the extra changes made earlier.
Fix: Revert extra changes